### ISSUE TITLE
05 model hyperparameters RandomizedSearchCV

### ISSUE THREAD

**Issue by srfrew** [UBC-MDS/fifa-potential#38]
I realized we're using hard coded values for `RandomizedSearchCV` with an explicit list of values instead of a distribution (like `loguniform()`. Would it be worth updating the script to do this? It may allow for RandomizedSearch to further refine performance.

**Comment by meretelutz** [comment]
I agree, we could definitely improve that. Along with the confusion matrix too. But I don't think its a big priority right now. Maybe we can tweak it after we submit this milestone?

**Comment by srfrew** [comment]
For sure, we can take a look after this milestone
